staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/converter_test.go

```diff
@@ -121,8 +121,7 @@ func doRoundTrip(t *testing.T, item runtime.Object) {
 		return
 	}
 
-	newUnstr := make(map[string]interface{})
-	err = DefaultConverter.ToUnstructured(item, &newUnstr)
+	newUnstr, err := DefaultConverter.ToUnstructured(item)
```

Comment:
Just have a naive question here: Is this returned value `newUnstr` in used ? if previous `ToUnstructured` doesn't return this value, can we just return only error as previously ?